machine_learning_in_r.Rmd

```diff
@@ -0,0 +1,133 @@
+---
+title: "R Machine Learning Workflow Tutorial"
+author: "Tongli Su(ts3478), Yifan Lu(yl5113)"
+output:
+  html_document:
+    fig_width: 5
+    fig_height: 3.5
+  pdf_document: default
+---

```

Comment:
Hi, please remove this YAML header and replace it with a header starting with a single hashtag and a single space, followed by a blank line, followed by the author's name.

<img width="685" alt="image" src="https://user-images.githubusercontent.com/6240731/202021077-c2982986-2e66-420e-87fa-7f6434ac9279.png">